src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -178,6 +181,11 @@ void function_call_harness_generatort::handle_option(
       }
     }
   }
+  else if(option == FUNCTION_HARNESS_GENERATOR_TREAT_POINTER_AS_CSTRING)
+  {
+    p_impl->function_parameters_to_treat_as_cstrings.insert(
+      values.begin(), values.end());
+  }
```

Comment:
The function is too long. It may already be the case, but your changes are making it worse.

Reply:
this is literally just a dispatch on name. Breaking this up into separate functions is unlikely to help with readability.

Reply:
it would be more readable if it would fit on one page